src/zenml/zen_stores/schemas/component_schemas.py

```diff
@@ -48,6 +67,7 @@ class StackComponentSchema(ShareableSchema, table=True):
     type: StackComponentType
     flavor: str
     configuration: bytes
+    metadata_values: Optional[bytes] = Field(sa_column=Column(HexByteString))
```

Comment:
I would call this simply `metadata` to use the same name as the model, which is how we usually handle these.

Reply:
Calling it `metadata` would cause a clash with a field by the same name in `SQLModel` and so I added the extra `_values` to it.